src/Lorentz/Contracts/BaseDAO.hs

```diff
@@ -19,21 +19,33 @@ import Lorentz
 import Lorentz.Contracts.BaseDAO.FA2
 import Lorentz.Contracts.BaseDAO.Management
 import Lorentz.Contracts.BaseDAO.Proposal
+import Lorentz.Contracts.BaseDAO.Doc (callFA2Doc)
 import Lorentz.Contracts.BaseDAO.Types as BaseDAO
 import qualified Lorentz.Contracts.Spec.FA2Interface as FA2
 
+-- | Common marker for permanent entrypoints.
+-- Can be used when parameter for permanent entrypoints is flat, i.e. does not
+-- have nested subparameters with multiple entrypoints.
+data PermanentEntrypointsKind
+instance DocItem (DEntrypoint PermanentEntrypointsKind) where
+  docItemPos = 1030
+  docItemSectionName = Just "Permanent entrypoints"
+  docItemToMarkdown = diEntrypointToMarkdown
```

Comment:
Probably you forgot to rename it? In our case, we need to use `PlainEntrypointsKind` for our top-level `entryCase` and a custom datatype for `entryCase` related to FA2 entrypoints.

Also, make sure that `docItemPos` is quite unique.

Reply:
I don't understand you. Do I need to create here a `PermanentEntrypointsKind` datatype and use it only in fa2Handler entryCase?

> Also, make sure that docItemPos is quite unique.
Is it calculated somehow, or just a random number?

Reply:
You need to create `FA2EntrypointsKind`, and use it only in fa2Handler entryCase, right.

> Is it calculated somehow, or just a random number?
It specifies global order on doc items in the rendered documentation. Just assign some similar number, like `1035`

Reply:
> Also, make sure that docItemPos is quite unique. 

Is it somehow calculated, or just a random number?
Also, can you explain further about `PermanetnEtrypointsKind` and the next comment